package.json

```diff
@@ -1,23 +0,0 @@
-{
```

Comment:
Olá, tudo bem? Irei fazer a análise do seu código... Qualquer dúvida que tiver a respeito da mesma, mande uma mensagem no slack que irei explicar da melhor maneira possível!

Lembrando que este pull request (PR) não é para ser aceito, você pode deletá-lo ou simplesmente ignorá-lo...

Resultado da revisão...

As mensagens de commit estão lindas...mas doeu meu coração ver vc usando .env e me deixando perdido sem um .env.example mostrando quais variáveis precisam ter no .env. Sempre que for usar .env, crie um .env.example que não fique no .gitignore, é através dele que saberei quais variáveis precisarei colocar no .env.